### PR TITLE
Implement RpcApi::list_since_block()

### PR DESCRIPTION
**Pull request by shesek**
Documented here: https://bitcoincore.org/en/doc/0.20.0/rpc/wallet/listsinceblock/

### CODE REVIEW

File: client/src/client.rs [hunk 1]
```diff
@@ -487,6 +487,22 @@ pub trait RpcApi: Sized {
...
+        include_watchonly: Option<bool>,
+        include_removed: Option<bool>,
+    ) -> Result<json::ListSinceBlockResult> {
+        let mut args = [
+            opt_into_json(blockhash)?,
+            opt_into_json(target_confirmations)?,
+            opt_into_json(include_watchonly)?,
+            opt_into_json(include_removed)?,
+        ];
+        self.call("list_since_block", handle_defaults(&mut args, &[null()]))
```
Comment (stevenroose) [inline client/src/client.rs:503]: I don't think this can be correct with these underscores.

Comment (shesek) [reply client/src/client.rs:503]: Oops, not sure how that slipped in... fixing

**Comment by shesek** [comment]
Fixed and added a simple test.

**Comment by R1971** [comment]
reufrujevic750@gmail.com
Blockonomics API key 2jtzgHdnTdEV4FjgcQxvwSNhmSM9mzqzswSkIpos6ks 
Verification

**Pull request merged**